torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1971,18 +1971,12 @@ c10::intrusive_ptr<c10::ivalue::Future> ProcessGroupNCCL::WorkNCCL::
 }
 
 float ProcessGroupNCCL::WorkNCCL::getDuration() const {
-  TORCH_CHECK(timingEnabled_, "getDuration only works if timing was enabled")
-  TORCH_CHECK(
-      ncclStartEvents_->size() == 1,
-      "getDuration only works for single device per ProcessGroup.");
-  TORCH_CHECK(
-      ncclEndEvents_->size() == 1,
-      "getDuration only works for single device per ProcessGroup.");
-  TORCH_CHECK(
-      (*ncclEndEvents_)[0].query(),
-      "getDuration can only be called after work is succeeded.")
-  return (*ncclStartEvents_)[0].elapsed_time((*ncclEndEvents_)[0]);
+  TORCH_CHECK(timingEnabled_, "getDuration only works if timing was enabled");
+  TORCH_CHECK(ncclStartEvents_, "getDuration only works if timing was enabled");
+  TORCH_CHECK(ncclEndEvents_, "getDuration only works if timing was enabled");
```

Comment:
nit: do you want to update the error msg here?

Reply:
i was intentionally lazy here. i think timing enabled --> start/end events should exist.  but i could make it more literally correct

Reply:
Well, I guess folks might want to change it later as a BE work anyway, lol.